src/index.ts

```diff
@@ -194,6 +194,17 @@ function setupOpenMRS() {
             moduleName
           }
         )
+      },
+      {
+        id: "capture-photo",
+        slot: "capture-patient-photo",
+        load: getAsyncLifecycle(
+          () => import("./widgets/attachments/capture-photo.component"),
+          {
+            featureName: "allergies-menu-item",
```

Comment:
The feature name should be "capture-photo-widget" or so, right?

Reply:
Nice catch, this was an oversight. Updated!